src/Output/Copy/Serializer/ContentType/ContentTypeSerializerInterface.php

```diff
@@ -7,7 +8,11 @@
 
 interface ContentTypeSerializerInterface
 {
+    public const LITERAL_VALUE_KEY = '__literalResponseValue';
```

Comment:
this is part of "/Copy/" and php 7.0 doesn't have visibility for consts if I'm not mistaken
```suggestion
    const LITERAL_VALUE_KEY = '__literalResponseValue';
```